Composite/Composite/MarkedComposite.fs

```diff
@@ -0,0 +1,13 @@
+namespace Composite
+
+open FSharp.Core
+
+[<RequireQualifiedAccess>]
+module MarkedComposite =
+
+    ///<summary>Initializes an empty marked composite.</summary>
+    ///<param name="mark">The mark.</param>
+    ///<typeparam name="TMark">The type of marks in the composite.</typeparam>
+    ///<typeparam name="TPayload">The type of payload objects in the composite.</typeparam>
+    let Empty<'TMark, 'TPayload> (mark: 'TMark) =
+        MComp.empty<'TMark, 'TPayload> mark
```

Comment:
May be we need to introduce a new assembly Composite.Interfaces and move to it wrappers like this one (CompositeExtensions, EnumerableExtensions). What do you think?

Reply:
Don't think so. Simply see no benefit in it whatsoever. I'd rather do away with the cryptic `C` which to me has virtually outlived its usefulness. The interface I'd rather have in c# for the functions therein contained would be as follows:
```
    Composite.Create<T>(IEnumerable<Composite<T>> components)
    Composite.CreateValue<T>(T value)
    MarkedComposite.Create<TMark, TPayload>(TMark mark, IEnumerable<Composite<TMark, TPayload>> components)
    MarkedComposite.CreateValue<TMark, TPayload>(TMark mark, TPayload value)
```

Reply:
Done.